org.codeberg.SHuRiKeN.Eeman.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id" : "sh.shuriken.Eeman",
```

Comment:
the manifest filename has to match the `id`.

Reply:
Oh nvm I will add a new one.